### DISCUSSION TITLE
supabase Invalid UTF-8 sequence error help!!!

### DISCUSSION THREAD

**Discussion by Minjeong79** [supabase/supabase#29452] [Questions]
Hello,

I’m reaching out regarding an urgent issue I’m facing. After creating a folder in storage and uploading images, I created a table to store the image URLs, which were displaying correctly. However, I suddenly started encountering the "Error: Invalid UTF-8 sequence" error on September 23, 2024.

I need assistance with resolving this error as soon as possible. What steps can I take to fix it? Thank you!

![문제1](https://github.com/user-attachments/assets/e7e5fa5f-d97a-4c0e-9823-dc9433609359)

**Comment by fincha** [comment]
same issue here... did you find a solution?
